src/file-uploads/file-uploads.service.ts

```diff
@@ -1,19 +1,25 @@
-import { Injectable } from "@nestjs/common";
+import { Injectable, HttpException, HttpStatus } from "@nestjs/common";
 
-import { S3Service } from "@/common/aws/s3-service/s3-service";
-import { PresignedUrlFileDto } from "@/file-uploads/file-uploads.dtos";
+import { FirebaseService } from "@/firebase/firebase.service";
+
+import { ALLOWED_MIME_TYPES } from "./file-uploads.constants";
 
 @Injectable()
 export class FileUploadsService {
-  constructor(private readonly s3Service: S3Service) {}
+  constructor(private readonly firebaseService: FirebaseService) {}
 
-  getPresignedUrl(fileUploadDto: PresignedUrlFileDto) {
-    return Promise.all(
-      fileUploadDto.files.map(async (file) => {
-        const { name, type } = file;
-        const signedUrl = await this.s3Service.getPresignedUrl(name, type);
-        return { name, type, signedUrl };
-      }),
-    );
```

Comment:
Please look into usage of pre signed url for file uploads

Reply:
Bhaiya, as I am using firebase as my storage, it does not go well with the pre signed url concept like S3. So I have implemented this way.

Reply:
I have another branch where I used S3 that was in the template but I was facing problem with docker and bucket, as soon as the docker is turned off I have to create the bucket again. It's not persisting, so I  have changed the storage system to firebase.

Reply:
There must be a way to do this for firebase as well. 
Please discuss with Ragib Bhai in the standup whether you should do that here or delegate that to another ticket.

Also, please mention the switch to firebase for storage.